impl/src/main/java/jaxrs/CollectionOptionsRequestParameterSerializer.java

```diff
@@ -0,0 +1,29 @@
+package jaxrs;
```

Comment:
Why is this and CollectionOptionsResolver needed? Looks to me like things that should work out of the box. Missing stuff in Reactive Wizard again?

Reply:
Yes.